doc/ApiDocGeneration/docindex.yml

```diff
@@ -0,0 +1,47 @@
+resources:
```

Comment:
I'm fine with moving the doc stuff under doc but lets keep the yml files under "eng/pipelines"

Reply:
Ok

Reply:
Done